packages/palette-docs/gatsby-config.js

```diff
@@ -33,10 +33,12 @@ module.exports = {
         // gatsby-mdx. See https://github.com/ChristopherBiscardi/gatsby-mdx/issues/243
         globalScope: `
           import * as Elements from "@artsy/palette"
+          import { ColorComponent } from "components/ColorComponent"
```

Comment:
We should keep the global scope clear unless changes are necessary; what is the purpose of this component compared to what was there before?

Reply:
It just encapsulates the structure that was already there. Cleaner if we were going to list out each individually. 

Doesn't strictly need to be a global, but it wasn't immediately clear to us the best way to pull things in.

Reply:
If it only exists singularly on the `colors.mdx` page (and will only ever exist there) then why break it out into its own component, etc?